data/en/test_radio_mandatory_with_mandatory_other.json

```diff
@@ -0,0 +1,48 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "radio",
+        "title": "RadioMandatory",
+        "blocks": [{
+                "type": "Questionnaire",
+                "id": "radio-mandatory",
+                "questions": [{
+                    "type": "General",
+                    "id": "radio-mandatory-question",
+                    "title": "Mandatory radio question, select an option",
+                    "answers": [{
+                            "type": "Radio",
+                            "id": "radio-mandatory-answer",
+                            "label": "What is you favourite breakfast item?",
+                            "mandatory": true,
+                            "options": [{
+                                "label": "Other",
+                                "description": "An answer is required.",
+                                "value": "Other",
+                                "child_answer_id": "other-answer-mandatory"
+                            }]
```

Comment:
All radio options should have at least two options. This isn't currently enforced in the schema but should be. This applies to all schemas in this pull request that have a single option.